scripts/zones/AlTaieu/mobs/Ulxzomit.lua

```diff
@@ -24,11 +24,13 @@ entity.onMobSpawn = function(mob)
     -- Baby Ul'xzomit
     if mob:getMobMod(xi.mobMod.LEADER) < 0 then
         mob:setMobFlags(1153)
+        -- MNK mob but do not attack twice
+        mob:delMod(xi.mod.MARTIAL_ARTS, 0)
```

Comment:
Shouldnt this be `setMod()`? `delMod()` means "remove/substract this amount"
So in this case, you are doing nothing (x - 0)

Reply:
Got that adjusted to setMod instead